src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -198,6 +199,9 @@ HARDWARE_INTRINSIC(Sve,           StoreNarrowing,
 HARDWARE_INTRINSIC(Sve,           StoreNonTemporal,                                                 -1,      3,      true,  {INS_sve_stnt1b,     INS_sve_stnt1b,     INS_sve_stnt1h,     INS_sve_stnt1h,     INS_sve_stnt1w,     INS_sve_stnt1w,     INS_sve_stnt1d,     INS_sve_stnt1d,     INS_sve_stnt1w,     INS_sve_stnt1d},  HW_Category_MemoryStore,           HW_Flag_Scalable|HW_Flag_BaseTypeFromFirstArg|HW_Flag_ExplicitMaskedOperation|HW_Flag_SpecialCodeGen|HW_Flag_LowMaskedOperation)
 HARDWARE_INTRINSIC(Sve,           Subtract,                                                         -1,      2,      true,  {INS_sve_sub,        INS_sve_sub,        INS_sve_sub,        INS_sve_sub,        INS_sve_sub,        INS_sve_sub,        INS_sve_sub,        INS_sve_sub,        INS_sve_fsub,       INS_sve_fsub},    HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_OptionalEmbeddedMaskedOperation|HW_Flag_HasRMWSemantics|HW_Flag_LowMaskedOperation)
 HARDWARE_INTRINSIC(Sve,           SubtractSaturate,                                                 -1,      2,      true,  {INS_sve_sqsub,      INS_sve_uqsub,      INS_sve_sqsub,      INS_sve_uqsub,      INS_sve_sqsub,      INS_sve_uqsub,      INS_sve_sqsub,      INS_sve_uqsub,      INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_OptionalEmbeddedMaskedOperation|HW_Flag_HasRMWSemantics|HW_Flag_LowMaskedOperation)
+HARDWARE_INTRINSIC(Sve,           TestAnyTrue,                                                      -1,      2,      true,  {INS_sve_ptest,      INS_sve_ptest,      INS_sve_ptest,      INS_sve_ptest,      INS_sve_ptest,      INS_sve_ptest,      INS_sve_ptest,      INS_sve_ptest,      INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation|HW_Flag_BaseTypeFromFirstArg|HW_Flag_SpecialCodeGen)
```

Comment:
Why does `Test*` needs SpecialCodeGen?

Reply:
2 things:
- Need to assert that the dst register is `REG_NA`
- I need to pass `INS_OPTS_SCALABLE_B` on `emitIns`

Reply:
that makes sense then.